cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/CountyCourtJudgment.java

```diff
@@ -1,14 +1,15 @@
-package uk.gov.hmcts.cmc.claimstore.models;
+package uk.gov.hmcts.cmc.domain.models;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
-import uk.gov.hmcts.cmc.claimstore.constraints.AgeRangeValidator;
-import uk.gov.hmcts.cmc.claimstore.constraints.DateNotInThePast;
-import uk.gov.hmcts.cmc.claimstore.constraints.Money;
-import uk.gov.hmcts.cmc.claimstore.constraints.ValidCountyCourtJudgment;
-import uk.gov.hmcts.cmc.claimstore.models.ccj.PaymentOption;
-import uk.gov.hmcts.cmc.claimstore.models.ccj.RepaymentPlan;
-import uk.gov.hmcts.cmc.claimstore.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.ccj.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.ccj.RepaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.constraints.AgeRangeValidator;
+import uk.gov.hmcts.cmc.domain.models.constraints.DateNotInThePast;
+import uk.gov.hmcts.cmc.domain.models.constraints.Money;
+import uk.gov.hmcts.cmc.domain.models.constraints.ValidCountyCourtJudgment;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.utils.ToStringStyle;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.gov.hmcts.cmc.domain.utils.ToStringStyle'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)